src/scss/skin-modern/components/_settingspanel.scss

```diff
@@ -68,6 +68,56 @@
       display: none;
     }
   }
+
+  .#{$prefix}-ui-subtitlesettingtoggle {
```

Comment:
Rename to `.#{$prefix}-ui-subtitlesettingsopenbutton` and extract to `_subtitlesettingsopenbutton.scss`